### PR TITLE
Add @geobde/use-click-away hook.

### PR DESCRIPTION
**Pull request by geobde**
React hook when you want a callback invoked when a DOM element was not clicked.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,3 +249,5 @@
 - [`useServiceWorker`](https://github.com/JCofman/react-hook-use-service-worker) A React hook which can register a service worker
 - [`useValueAfter`](https://github.com/bboydflo/use-value-after) Very simple React hook to easily provide different props to a component (comes in handy for testing edge cases)
 - [`useScreenType`](https://github.com/wednesday-solutions/react-screentype-hook) React hook to dynamically get current screen type (mobile, tablet, desktop) with configurable breakpoint support.
+- [`use-click-away`](https://github.com/geobde/use-click-away) React hook when you want a callback invoked when a DOM element was not clicked.
```
Comment (aarongarciah) [inline README.md:252]: Can you sort it alphabetically?

Comment (geobde) [reply README.md]: i did it!

**Comment by aarongarciah** [comment]
Once conflicts are resolved this can be merged 😄

File: README.md [hunk 2]
```diff
@@ -251,4 +252,5 @@
 - [`useScrollSpy`](https://github.com/Purii/react-use-scrollspy) React hook to automatically update navigation based on scroll position.
 - [`useServiceWorker`](https://github.com/JCofman/react-hook-use-service-worker) A React hook which can register a service worker
 - [`useValueAfter`](https://github.com/bboydflo/use-value-after) Very simple React hook to easily provide different props to a component (comes in handy for testing edge cases)
+- [`useScreenType`](https://github.com/wednesday-solutions/react-screentype-hook) React hook to dynamically get current screen type (mobile, tablet, desktop) with configurable breakpoint support.
```
Comment (aarongarciah) [inline README.md:255]: This line is duplicated (see line 251)

Comment (geobde) [reply README.md]: i fix it.

**Comment by aarongarciah** [comment]
Thanks!

**Pull request merged**